SCClassLibrary/Common/Control/Recorder.sc

```diff
@@ -100,11 +100,16 @@ Recorder {
 
 	prepareForRecord { | path, numChannels |
 		var bufSize = recBufSize ? server.recBufSize ?? { server.sampleRate.nextPowerOfTwo };
+		var dir;
 
 		recHeaderFormat = recHeaderFormat ? server.recHeaderFormat;
 		recSampleFormat = recSampleFormat ? server.recSampleFormat;
 		numChannels = numChannels ? server.recChannels;
+
 		path = if(path.isNil) { this.makePath } { path.standardizePath };
+		dir = path.dirname;
```

Comment:
in the future (or now if you like), it would be good to refactor this into `String`. For so long, we have had no simple method that would just simply guarantee that a path exists and is well formatted. Maybe this could even be done in `standardizePath`?

Reply:
> For so long, we have had no simple method that would just simply guarantee that a path exists and is well formatted.

What do you mean? Doesn't `File.exists` accomplish this? Also, that is definitely outside the scope of this PR.